subsquid/src/pabloProcessor.ts

```diff
@@ -59,8 +59,8 @@ interface PoolCreatedEvent {
 }
 
 function getPoolCreatedEvent(event: PabloPoolCreatedEvent): PoolCreatedEvent {
-    if (event.isV2100) {
-        const {owner, poolId, assets} = event.asV2100;
```

Comment:
Do we need to remove the old parsing logic?

Reply:
@saadjhk what do you think about this?

Reply:
I think for now removing old parsing logic is ok as the runtime is yet to be released to a production chain that we need to keep backwards compatibility with.